prime-router/src/main/kotlin/cli/main.kt

```diff
@@ -283,8 +301,28 @@ class ProcessData : CliktCommand(
             }
         }
 
+        // synthesize the data here
+        // todo: put these strategies into metadata so we can load them from a file
+        val synthesizeStrategies = mapOf(
+            "patient_last_name" to Report.SynthesizeStrategy.SHUFFLE,
+            "patient_first_name" to Report.SynthesizeStrategy.SHUFFLE,
```

Comment:
I'm not seeing a reason to ever shuffle real names - let's just always make them fake.

Date of Birth, on the other hand, might be valuable to fake the day and month, but keep the year, so the data has realistic age shape.